src/apis/teams/api.ts

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+
+import * as usersApi from 'apis/users';
+import {APIArrayResponse} from 'types/api';
+import {CoreTeamMember, Team, CoreTeam} from 'types/teams';
+import {GetCoreTeamResponse, GetCoreTeamMemberResponse, GetTeamMemberResponse} from './types';
+
+export async function getTeamById(id: string): Promise<Team> {
+  const response = await axios.get(`${process.env.REACT_APP_BACKEND_API}/teams/${id}`);
+  return response.data ? response.data : null;
+}
+
+export async function getCoreTeams(): Promise<CoreTeam[]> {
+  const response = await axios.get<APIArrayResponse<GetCoreTeamResponse>>(
+    `${process.env.REACT_APP_BACKEND_API}/core_teams`,
+  );
+
+  if (!response.data) {
+    return [];
+  }
+
```

Comment:
@jamessspanggg just curious but if something goes wrong here, how do we show the appropriate error to the user?

Since you're sending an empty array as a response, it could also mean API worked fine but there were no core teams to be fetched.